src/lib/models/types/intrinsic.ts

```diff
@@ -14,6 +14,11 @@ export class IntrinsicType extends Type {
     name: string;
 
     /**
+     * The type name identifier.
+     */
+    readonly type: string = 'instrinct'; // TODO: Is there any change to correct this typo?
```

Comment:
What do you think we should do here? I'm happy to accept the fix in a future PR. Just let me know what you think is needed to fix this!